templates/install/project.md

```diff
@@ -104,22 +104,30 @@ Also, you can get the Lean documentation inside VS Code using
 inside the text field that appears, type "lean doc" and hit <kbd>Enter</kbd>.
 Then click "Mathematics in Lean" or "Theorem Proving in Lean" and enjoy.
 
-## Hosting your project on GitHub
+## Updating Mathlib in your project
 
-Your project is already a git repository, and as it grows,
-you may want to host it on [GitHub](https://guides.github.com/activities/hello-world/).
-If you take this step, the community offers some
-[GitHub Actions scripts](../ci.html) that could help manage your repository.
-But don't worry if you don't know what this means.
-It's not necessary for using Lean.
+If you have a project depending on Mathlib, and you want to update to the latest version of Lean and Mathlib, you have to do two steps:
+* Update the `lean-toolchain` file in your repository to the latest version. You can do this automatically by running the following in the root directory of your repository.
+```
+curl https://raw.githubusercontent.com/leanprover-community/mathlib4/master/lean-toolchain > lean-toolchain
+```
+* run `lake update`. This will update Lean, Mathlib and download the new Mathlib cache for you.
```

Comment:
It's preferable to use `lake -R -Kenv=dev update`. `-R` avoids lake to use an old lakefile.olean. `-Kenv=dev` avoids removing doc-gen4 from the manifest if the user is using it.

See discussions on Zulip [1](https://leanprover.zulipchat.com/#narrow/stream/113489-new-members/topic/Recommend.20way.20to.20add.20mathlib.20master.20as.20lib.3F/near/438097875), [2](https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/UnicodeBasic.20error.20when.20upgrading.20to.204.2E8.2E0rc2/near/441776050).

Reply:
Thanks for the pointers. 
I missed the `-R` option in my own repositories using doc-gen, so thanks for that piece of information. These instructions are not really for using doc-gen, but I guess adding the general case doesn't hurt.